src/style/colors.ts

```diff
@@ -162,7 +163,8 @@ export class Colors {
     return validColors ? undefined : results[0];
   }
 
-  getColorTint(colorValue: string | OpaqueColorValue, tintKey: string | number) {
+  getColorTint(colorValue: string | OpaqueColorValue, tintKey: string | number, options?: GetColorTintOptions) {
+    const avoidReverseOnDark = !!options?.shouldReverseOnDark;
```

Comment:
I think you misunderstood me here.. 
What I meant is that the option name will be avoidReverseOnDark, this way you don't need to do any special manipulation, and if the user passes nothing it will behave as we meant to